.babelrc

```diff
@@ -1,7 +1,7 @@
 {
     "presets": [
         "@babel/env",
-        "@babel/typescript"
+        "@babel/preset-typescript"
```

Comment:
It is not necessary to use "preset" keyword -  [preset-shorthand](https://babeljs.io/docs/en/presets#preset-shorthand).

`"@babel/preset-typescript"` and `"@babel/typescript"` both are valid names.

Reply:
you're right.